www/src/views/shared/thumbnail.js

```diff
@@ -60,8 +60,8 @@ export default ThumbnailLink
 
 export const showcaseThumbnailFragment = graphql`
   fragment ShowcaseThumbnailFragment_item on ImageSharp {
-    fixed(width: 282, height: 211) {
-      ...GatsbyImageSharpFixed_noBase64
+    fluid(maxWidth: 350) {
```

Comment:
Ries, I missed one thing here earlier—if you search for starters by @LeKoArts (👋) you'll see that we need to crop the images. You can do so for fluid images by supplying a [`maxHeight`](https://next.gatsbyjs.org/packages/gatsby-image/?=gatsby-image#query-1). Oh, and setting [`cropFocus`](https://www.gatsbyjs.org/packages/gatsby-plugin-sharp/?=gatsby-plugin-sharp#cropfocus) to `NORTH` should prevent this happening:

![image](https://user-images.githubusercontent.com/21834/45259901-3cd99380-b3d8-11e8-8c14-5899f29fc2f4.png)